src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,49 @@
+package duke.task;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.DukeException;
+
+/**
+ * Event represents Event tasks.
+ */
+public class Event extends Task {
+
+    protected LocalDate at;
```

Comment:
Variable name can be more descriptive
```suggestion
    protected LocalDate atDateTime;
```